### PR TITLE
Update sidecar-injection.md

### PR DESCRIPTION
**Pull request by andraxylia**
Removed the trailing $ in front of commands so they can be copy pasted.
Fixed the label deletion command.

### CODE REVIEW

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ kubectl apply -f <(~istioctl kube-inject -f samples/sleep/sleep.yaml)
 Verify that the sidecar has been injected into the deployment.
 
 ```
-$ kubectl get deployment sleep -o wide
+kubectl get deployment sleep -o wide
```
Comment (ayj) [inline _docs/setup/kubernetes/sidecar-injection.md]: Please move the command output to a separate block here and elsewhere.

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 2]
```diff
@@ -176,28 +176,28 @@ $ ./install/kubernetes/webhook-create-signed-cert.sh \
...
 ```
-$ kubectl apply -f install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
+kubectl apply -f install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
 ```
 
 The sidecar injector webhook should now be running.
 
 ```
-$ kubectl -n istio-system get deployment -listio=sidecar-injector
+kubectl -n istio-system get deployment -listio=sidecar-injector
```
Comment (ayj) [inline _docs/setup/kubernetes/sidecar-injection.md]: Output needs to be in a separate block

**Comment by andraxylia** [comment]
@ldemailly can you give @ayj reviewer permission on docs?

File: _docs/setup/kubernetes/sidecar-injection.md (hunk 1 above)
Comment (andraxylia) [reply _docs/setup/kubernetes/sidecar-injection.md]: Fixed in a subsequent PR, not sure how to update one created from "Edit this file"

Comment (ldemailly) [reply _docs/setup/kubernetes/sidecar-injection.md]: click edit when in your branch

**Comment by ldemailly** [comment]
I just checked jason is already in doc hackers

**Pull request merged**